lib/index.js

```diff
@@ -627,25 +781,133 @@ class NewsArticle extends BaseAsset {
         this._authors = [];
     }
 
-    // Takes a string or array of strings
+    /**
+     * Pass an array of names of article authors. If `value` is a string
+     * instead of an array, it is assumed there is only one author. For the
+     * blog article format, see {@link BlogArticle#set_author}.
+     * @param {(string|string[])} value - The name or names of the author(s)
+     */
     set_authors(value) {
         this._authors = value;
         if (!(value instanceof Array))
             this._authors = [value];
     }
 
+    /**
+     * Marks an image as the "main" image of the article, which is marked up
+     * specially. Includes an optional caption. Compare
+     * {@link BlogArticle.set_main_image} and
+     * {@link BlogArticle.set_main_image_caption}.
+     *
+     * (Make sure to remove this image from the HTML that you pass to
+     * {@link NewsArticle#set_body}, otherwise it will be present twice in the
+     * rendered post.)
+     *
+     * @param {ImageAsset} asset - The main image
+     * @param {(string|Cheerio)} [caption] - HTML for the main image's caption
+     */
     set_main_image(asset, caption) {
         this._main_image = asset;
         this._main_image_caption = _ensure_cheerio(caption);
     }
 
     set_tags(value) { throw new Error(`Use set_section() for NewsArticle`); }
+    /**
+     * Pass in the ID string of a "set" (category, tag). This is not the
+     * human-readable string (though it's possible for the two to be
+     * identical); the human-readable name of the set is created later in your
+     * permanent shard.
+     *
+     * (Compare {@link BlogArticle#set_tags} for the blog article format; news
+     * articles are usually in one section of the news site at a time. If it
+     * turns out we need more tags, we can also add a `set_tags()` method to
+     * the news article format.)
```

Comment:
This last sentence looks like something we should put in a `TODO` comment in the code, rather in the documentation block.